query-github.sh

```diff
@@ -89,16 +89,17 @@ for i in $(seq "${page_count}"); do
     #    don't care about for the same reasons.
     grep "        \"html_url" < "${curl_output_file}" \
         | grep -v "          " \
-        | sort | uniq \
+        | sort -u \
         | cut -d \" -f 4 \
         | grep -v "kelloggm" \
```

Comment:
Please don't hard-code these here.  Instead, put them in the query itself.
Hard-coding them has negative consequences:
 * Users have no way of knowing how to do the filtering.
 * Once users figure it out, they must edit the script.
 * Doing the filtering at GitHub means more useful results.  If half of results are these duplicates, then the current approach can return at most 500 useful results.  If the filtering is done at GitHub, then all 1000 search results that get to this point are useful, which means that users can run bigger experiments.

 If there are 1000 hits, and half of them are junk, its' as if

Reply:
You're correct. I missed your comment about this in the previous review - my mistake! I've removed these hard-coded filters, and added those that can be included in a GitHub search query (the ones about the owner) to the example query in this repo.

Preventing GitHub from returning hard forks of libcore, apache-harmony, or similar projects I think we should leave for future work.